sw/device/tests/sim_dv/meson.build

```diff
@@ -0,0 +1,672 @@
+# Copyright lowRISC contributors.
```

Comment:
Nit. Meson has been removed from the src tree, so I would drop this changes before rebasing.

Reply:
This file is supposed to be automatically removed. I think we don't need to change it as long as all tests are passing. Let's see how it goes.